buildkite/production-deploy-pipline.yml

```
@@ -0,0 +1,12 @@
+env:
+  HINTR_BRANCH: "${HINTR_BRANCH:-master}"
+  HINT_BRANCH: "${HINT_BRANCH:-master}"
+  DEPLOY: "${DEPLOY:-all}"
+
+
+steps:
+  - label: ":rocket: production"
+    command: "ssh -i ~/.ssh/id_hiv -oStrictHostKeyChecking=accept-new vagrant@naomi.dide.ic.ac.uk './clean-and-deploy-naomi.sh --hintr-branch=$HINTR_BRANCH --hint-branch=$HINT_BRANCH $DEPLOY'"
```

```suggestion
    command: >-
      ssh -i ~/.ssh/id_hiv -oStrictHostKeyChecking=accept-new vagrant@naomi.dide.ic.ac.uk 
      './clean-and-deploy-naomi.sh --hintr-branch=$HINTR_BRANCH --hint-branch=$HINT_BRANCH $DEPLOY'
```

for lekan's suggestion, this is I think the correct yaml incantation